lib/tasks/creator_seed.rake

```diff
@@ -1,17 +1,69 @@
 # frozen_string_literal: true
 require "csv"
-namespace :vocab do
+namespace :creators do
+  desc "Collects creator names and saves them to csv"
+  task collect: :environment do
+    creators_path = Rails.root.join("data", "creators.csv")
+
+    solr = Blacklight.default_index.connection
+
+    response = solr.get 'select', params: { "facet.field": ["creator_sim"], "facet.limit": -1 }
+    creators_array = response["facet_counts"]["facet_fields"]["creator_sim"]
+
+    CSV.open(creators_path, 'w', write_headers: true, headers: ["display_name", "result_count"]) do |row|
+      creators_array.each_slice(2) do |pair|
+        row << pair
+      end
+    end
+    # Same as above, one-line syntax
+    # CSV.open(creators_path,'w', :write_headers=> true, :headers => ["display_name", "result_count"]) { |row| creators_array.each_slice(2) { |pair| row << pair } }
+  end
+
   desc "Create database-based local authorities"
   task create: :environment do
     creators_path = Rails.root.join("data", "creators.csv")
-    creators = CSV.read(creators_path, headers: false)
+    creators = CSV.read(creators_path, headers: true)
     creators.each do |row|
-      Creator.where(display_name: row.first).first_or_create do |creator|
-        creator.display_name = row.first
+      Creator.where(display_name: row["display_name"]).first_or_create do |creator|
+        creator.display_name = row["display_name"].strip
         puts "Authority entry created #{creator.display_name}"
       end
     end
 
     puts Creator.count.to_s
```

Comment:
Add a little more context here - I wouldn't know what this number means without reading the code
```
puts "#{Creator.count.to_s} creators processed and added or updated."
```